### ISSUE TITLE
Unable to debug IronPython projects using Visual Studio 2017, Version 15.5.4

### ISSUE THREAD

**Issue by csanin2016** [Microsoft/PTVS#3583]
Hi everyone,  I am not able to debug an IronPython project in Visual Studio 2017, Version 15.5.4. The IronPython (.NET) launcher is selected as my Launch mode in the Debug project properties.. I am not able to get the IDE to stop at any break point. when using the 2.7 Python Environment, which is the only one that supports IronPython projects. Does any one knows what else I can try? Thanks!

**Comment by zooba** [comment]
To clarify, is your Python environment labelled "IronPython 2.7" or "Python 2.7"? It's not clear from your description.

If you are using "Python 2.7", then it is being treated as normal Python and so IronPython debugging won't work. I thought we popped up an error in this case, but perhaps not.

If you are definitely using IronPython, could you give us the exact version number (e.g. 2.7.5, 2.7.7)? Thanks

**Comment by csanin2016** [comment]
Thanks so much for your reply. I just found out that this happened  when I downloaded a zip file that placed the IronPython environment in a directory structure without being installed properly.  I removed this installation, and used the MSI file to install it. After this, I was able to debug properly using Visual Studio IDE configuring IronPython 2.7 (32 bit) in the solution.  Thanks again for trying to help!
